sdk/README.md

```diff
@@ -1,4 +1,45 @@
-## Contribute to SDK
-KFP v2 for SDK is under development in the main branch. 
+## Contributing to the `kfp` SDK
+KFP v2 for SDK is under development on the `master` branch.
 
 To contribute to v1, please go to [sdk/release-1.8 branch](https://github.com/kubeflow/pipelines/tree/sdk/release-1.8) to submit your change. 
+
+For general contribution guidelines including pull request conventions, see [pipelines/CONTRIBUTING.md](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md).
+
+### Code Style
+
+#### Style Guide [Required]
+We follow the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html).
+
+#### Formatting [Required]
+Please format your code using [yapf](https://github.com/google/yapf) according to the [`.style.yapf`](https://github.com/kubeflow/pipelines/blob/master/.style.yapf) file.
+
+From the project root, run the following code to format your code:
+```sh
+docker container run \
+  -v $PWD:/root/pipelines \
+  python:3.7-slim \
+  /bin/bash -c "pip install yapf && yapf --in-place --recursive /root/pipelines/sdk/python"
```

Comment:
Should we just give the `yapf` command? Every KFP SDK developer must have Python environment already, installing `yapf` via pip isn't much extra work (we could even include it in some `requirements-dev.txt` for example). On the other hand, they may not necessarily have docker runtime installed or running all the time.

Reply:
Same argument for `pylint` and `docformatter`

Reply:
Thanks for the suggestion, I agree.

Jointly, [70076a2](https://github.com/kubeflow/pipelines/pull/7436/commits/70076a2d50dcc23a67ee365ddab551f6e056a625), [d5b1364](https://github.com/kubeflow/pipelines/pull/7436/commits/d5b136479320d6b97e23b442d899df5c43e63d57), and [ddc6d41](https://github.com/kubeflow/pipelines/pull/7436/commits/ddc6d413718cf04c3852dd3a19f6b7ec0cce727e) achieve this change.